### PR TITLE
Remove unlisted-addons and automatic-validation waffles

### PR DESCRIPTION
**Pull request by mstriemer**
r? @magopian

### CODE REVIEW

File: apps/devhub/templates/devhub/addons/submit/upload.html [hunk 1]
```diff
@@ -21,41 +19,37 @@
...
-  {% endif %}
+  </div>
   <input type="file" id="upload-addon"
     data-upload-url="{{ url('devhub.upload') }}"
-    {% if waffle.flag('unlisted-addons') %}
       data-upload-url-listed="{{ url('devhub.upload') }}"
       data-upload-url-unlisted="{{ url('devhub.upload_unlisted') }}"
-      data-upload-url-sideload="{{ url('devhub.upload_sideload') }}"
-    {% endif %}>
+      data-upload-url-sideload="{{ url('devhub.upload_sideload') }}">
```
Comment (magopian) [inline apps/devhub/templates/devhub/addons/submit/upload.html]: dedent?

File: apps/devhub/templates/devhub/validate_addon.html [hunk 2]
```diff
@@ -40,42 +38,38 @@
...
+    </div>
 
     <input type="file" id="upload-addon"
       data-upload-url="{{ url('devhub.standalone_upload') }}"
-      {% if waffle.flag('unlisted-addons') %}
         data-upload-url-listed="{{ url('devhub.standalone_upload') }}"
         data-upload-url-unlisted="{{ url('devhub.standalone_upload_unlisted') }}"
-        data-upload-url-sideload="{{ url('devhub.standalone_upload_sideload') }}"
-      {% endif %}>
+        data-upload-url-sideload="{{ url('devhub.standalone_upload_sideload') }}">
```
Comment (magopian) [inline apps/devhub/templates/devhub/validate_addon.html]: dedent here too

File: apps/devhub/tests/test_views.py [hunk 3]
```diff
@@ -3001,10 +2934,8 @@ def test_listed_beta_pass_validation_with_flag(self, mock_sign_file):
         assert log.action == amo.LOG.BETA_SIGNED_VALIDATION_PASSED.id
 
     @mock.patch('devhub.views.sign_file')
-    def test_listed_beta_do_not_pass_validation_with_flag(self,
-                                                          mock_sign_file):
+    def test_listed_beta_do_not_pass_validation(self, mock_sign_file):
         """Beta files that don't pass validation should raise an error."""
```
Comment (magopian) [inline apps/devhub/tests/test_views.py]: Not related to this PR, but just realized the docstring is wrong: it shouldn't raise an error, it should simply log that it failed validation. Do you think you could fix that at the same time?

**Comment by magopian** [comment]
This looks really good! There's a few nits, and a failing test.

r+wc

Please, when resolving the bug as "fixed", make a note to the QA team to thoroughly test the add-on submission flow (for listed and unlisted), the standalone validator, uploading a new version, and uploading a new file (for listed and unlisted).
Those are the scenarios we had the more regressions with.

**Comment by mstriemer** [comment]
Tag is done so I'll merge this. Just added the bug number to my commit but the [build passed](https://travis-ci.org/mozilla/olympia/builds/86440425) previously.

**Pull request merged**